BadCodeV4.php

```
@@ -0,0 +1,70 @@
+
+class userController {  // 类名应该是 UserController (大写)
+    private $db;
+    public $users;  // 不应该是 public
+
+    // 构造函数缺少参数类型声明
+    function __construct($database) {
+        $this->db = $database;
+    }
+
+    // 方法名不规范，参数缺少类型声明
+    public function getUserById($userId, $includeProfile = null) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$includeProfile'. (Unused Code Rules)

(UnusedFormalParameter)